catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -93,6 +123,62 @@ public void init() {
 
     }
 
+    /**
+     * returns a Map of the parsed Accept-Encoding header according to RFC-2616-14.2
+     * </br>
+     * https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html
+     *
+     * @param header Accept-Encoding header to parse
+     * @return Map of {Encoding, Q value}
+     */
+    private Map<String, String> parseAcceptEncodings(String header) {
+        if (StringUtils.isEmpty(header)) {
+            return Collections.emptyMap();
+        }
+        String[] encodings = commaSpace.split(header.trim());
+        return Arrays.stream(encodings)
+                .map(semicolon::split)
+                .collect(Collectors.toMap(v -> v[0],
```

Comment:
I don't believe so because at this point we have confirmed the header is not empty (line 135) and the Pattern `split` method will always return an array with a single item in it even if there is no delimiter token found in the string. 

http://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html#split(java.lang.CharSequence,%20int)
>  If this pattern does not match any subsequence of the input then the resulting array has just one element, namely the input sequence in string form.